src/test/java/com/revature/servicestests/SearchServiceTest.java

```diff
@@ -241,10 +241,11 @@ void searchServiceOverallFilter()
 		String title = content.getTitle();
 		String format = content.getFormat();
 		List<Integer> mlist = new ArrayList<Integer>();
+		List<String> tempArr = new ArrayList<String>();
 		mlist.add(module1.getId());
 		mlist.add(module2.getId());
-
-		Set<Content> filtered = ss.filter(title, format, mlist);
+		//tempArr IS TEMPORARY TO SILENCE ERRORS 
```

Comment:
delete comment

Reply:
Comment necessary to indicate purpose of temporary variable